x-pack/test_serverless/functional/test_suites/common/data_view_mgmt.ts

```diff
@@ -100,5 +100,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await testSubjects.missingOrFail('rollup-tag');
       });
     });
+    it('Sample data loads', async () => {
```

Comment:
Could we have this test added to a separate test file (e.g. `sample_data.ts`) next to this file? It is accessed _through_ data views but is not a feature of data views.

Reply:
[a1cb601](https://github.com/elastic/kibana/pull/165157/commits/a1cb6010e087580a618d7b9b58203bf6ee7a1107) for changes thanks!!!